roles/test_snmpTraps/meta/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+galaxy_info:
+  author: CloudOps
+  description: "Tests for the snmpTraps functionality deployed in STF"
```

Comment:
This should include a reference to the validation procedures that define these tests

Reply:
@elfiesmelfie You mean link to the documentation ? Do we have it in the alerts test ? I copied the meta/main.yml from alerts test that you did as an example and changed the description.